utils.js

```diff
@@ -0,0 +1,57 @@
+const chalk = require('chalk');
```

Comment:
If you are going to use it for simple colors. console.log() already supports the use of different colors: like this:
colors.js
```
module.exports = {
  red: '\x1b[31m%s\x1b[0m',
  green: '\x1b[32m%s\x1b[0m',
  blue: '\x1b[34m%s\x1b[0m'
};
```


`console.log(red, 'Required parameters are: tech, category, name');`